openshift-templates/hawkular-apm-server.yml

```diff
@@ -8,8 +8,8 @@ metadata:
     description: Hawkular APM Server service.
     iconClass: icon-wildfly
     openshift.io/display-name: Hawkular APM Server
-    tags: instant-app,hawkular,apm,opentracing
-  name: hawkular-apm-server
+    tags: instant-app,hawkular,apm,opentracing,quickstart
```

Comment:
+1 , I changed because the "latest" version of Openshift has changed the UI for the templates screen, and people will now have to click "Uncategorized" to find the APM instant app :( I tried adding `quickstart` to see how it would influence the display on the UI, but forgot to change it back.